packages/compress/zlib/package.mk

```diff
@@ -36,6 +36,16 @@ PKG_AUTORECONF="no"
 TARGET_CONFIGURE_OPTS="--prefix=/usr"
 HOST_CONFIGURE_OPTS="--prefix=$ROOT/$TOOLCHAIN"
 
+post_configure_target() {
+ ## configure minizip
+ cd $ROOT/$PKG_BUILD/.$TARGET_NAME/contrib/minizip
+ rm Makefile
+ export PKG_CONFIG_PATH="$PKG_CONFIG_PATH:../../"
+ do_autoreconf
+ ./configure --host=$TARGET_NAME --build=$HOST_NAME $TARGET_CONFIGURE_OPTS
```

Comment:
--disable-shared --enable-static if you are going to use my post_makeinstall_target below